### DISCUSSION TITLE
账号被误封

### DISCUSSION THREAD

**Discussion by 120090162** [BurnHair/docs#151] [技术支持]
`用户6884已被封禁：重复注册 会话重复 (request id: 20240707063340725741954uSNKCSdW) (request id: 2024070706333986955886973020805)`
我没有重复注册过，有用tg每天签到攒额度，有几天没用，今天突然发现被封，但我只在one-api部署过，也只是我自己用，能给出原因解封吗

**Comment by BennyThink** [comment]
可以了。
